notifier.rst

```diff
@@ -436,7 +437,7 @@ Service          Package                               DSN
 
 .. versionadded:: 6.3
 
-    The PagerDuty and Pushover integrations were introduced in Symfony 6.3.
+    The Ntfy, PagerDuty and Pushover integrations were introduced in Symfony 6.3.
```

Comment:
Please add a 6.4 block, as we are in feature freeze phase right now.

Reply:
@OskarStark : yes, I will review it when the 6.4 branch will be ready

Reply:
It is ready now